src/components/Advertisements/core/AdvertisementEntry/AdvertisementEntry.test.tsx

```diff
@@ -308,43 +308,6 @@ describe('Testing Advertisement Entry Component', () => {
     });
   });
 
-  test('Updates the advertisement and shows error toast on successful update', async () => {
-    const updateAdByIdMock = jest.fn();
-
-    mockUseMutation.mockReturnValue([updateAdByIdMock]);
-
-    render(
-      <ApolloProvider client={client}>
-        <Provider store={store}>
```

Comment:
Why was this removed?

Reply:
this test was redundant and was causing problems while testing. Removing it did not have an impact on the branch and line coverage.